docs/source/en/testing.md

```diff
@@ -511,15 +511,20 @@ from transformers.testing_utils import get_gpu_count
 n_gpu = get_gpu_count()  # works with torch and tf
 ```
 
-### Testing with a specific PyTorch backend
+### Testing with a specific PyTorch backend or device
 
-To run the test suite on a specific torch backend add `TRANSFORMERS_TEST_DEVICE="$device"` where `$device` is the target backend. For example, to test on CPU only:
+To run the test suite on a specific torch device add `TRANSFORMERS_TEST_DEVICE="$device"` where `$device` is the target backend. For example, to test on CPU only:
 ```bash
 TRANSFORMERS_TEST_DEVICE="cpu" pytest tests/test_logging.py
 ```
 
 This variable is useful for testing custom or less common PyTorch backends such as `mps`. It can also be used to achieve the same effect as `CUDA_VISIBLE_DEVICES` by targeting specific GPUs or testing in CPU-only mode.
 
+Certain devices will require an additional import after importing `torch` for the first time. This can be specified using the environment variable `TRANSFORMERS_TEST_BACKEND`:
+```bash
+TRANSFORMERS_TEST_BACKEND="torch_npu" pytest tests/test_logging.py
```

Comment:
`tests/test_logging.py` doesn't exist in `transformers`.

It would be nice if the example test against a test file (or even a single test device) that could use the `npu` device with the necessary backends. But I guess such a test doesn't exist yet and will wait your work on #25654.

Let keep this in mind (to update later)

Reply:
> tests/test_logging.py doesn't exist in transformers.

Interesting, it is referenced numerous times in the `docs/source/en/testing.md` file!